assets/blocks/quiz/index.js

```diff
@@ -0,0 +1,45 @@
+import { InnerBlocks } from '@wordpress/block-editor';
```

Comment:
Should we name the directory quiz-block?

Reply:
Updated everything to the new conventions in c8455e3.
I left a top-level `quiz` folder to group things, and added the quiz block itself to `quiz/quiz-block`.